src/stim/stabilizers/tableau.pybind.cc

```diff
@@ -208,6 +208,34 @@ void stim_pybind::pybind_tableau(pybind11::module &m) {
         )DOC")
             .data());
 
+    c.def(
+        "to_pauli_string",
+        [](const Tableau &self) {
+            return PyPauliString(self.to_pauli_string());
+        },
+        clean_doc_string(u8R"DOC(
+            If the tableau fixes each pauli upto sign, creates the pauli
+            string which represents the tableau.
+
+            Returns:
+                The created pauli string
+
+            Raises:
+                ValueError: The Tableau isn't equivalent to a Pauli product.
+
+            Example:
+                >>> import stim
+                >>> p = stim.PauliString("+YX_Z")
+                >>> t = p.to_tableau()
+                >>> t.to_pauli_string()
+                stim.PauliString("+YX_Z")
+                >>> cnot = stim.Tableau.from_named_gate("CNOT")
+                >>> cnot.to_pauli_string()
```

Comment:
I generally avoid showing errors in examples, just because examples are things for people to try and have it work and generally people aren't trying to cause an error.

It's definitely a good *test* to have, though!

Reply:
makes sense